src/NvimClient.APIGenerator/NvimAPIGenerator.cs

```diff
@@ -123,11 +123,28 @@ private object GetExtensionType(MessagePackExtendedTypeObject msgPackExtObj)
       string.Join("\n",
         uiEvents.Select(uiEvent =>
         {
-          var camelCaseName = StringUtil.ConvertToCamelCase(uiEvent.Name, true);
+          var camelCaseName = $"{StringUtil.ConvertToCamelCase(uiEvent.Name, true)}Event";
```

Comment:
```suggestion
          var camelCaseName = $"{StringUtil.ConvertToCamelCase(uiEvent.Name, true)}UIEvent";
```

or...

```suggestion
          var camelCaseName = $"UI{StringUtil.ConvertToCamelCase(uiEvent.Name, true)}Event";
```

Reply:
Note that if I do this I’ll have to modify a few other parts of the code, since this changes the name structure of all the events entirely (from what #11 made them). 

I’m fine with doing this if you think it’s a good idea (although there’s something I dislike about all the events having `UIEvent` at the end of their names, when we could just do `api.UIEvents.Put` etc. instead), but it’s kinda outside the scope of this PR. Maybe better suited to a follow-up one?